models/refresh_token.go

```diff
@@ -87,16 +88,15 @@ func GetValidChildToken(tx *storage.Connection, token *RefreshToken) (*RefreshTo
 }
 
 // Logout deletes all refresh tokens for a user.
-func Logout(tx *storage.Connection, instanceID uuid.UUID, id uuid.UUID) error {
-	return tx.RawQuery("DELETE FROM "+(&pop.Model{Value: RefreshToken{}}).TableName()+" WHERE instance_id = ? AND user_id = ?", instanceID, id).Exec()
+func Logout(tx *storage.Connection, id uuid.UUID) error {
+	return tx.RawQuery("DELETE FROM "+(&pop.Model{Value: RefreshToken{}}).TableName()+" WHERE instance_id = ? AND user_id = ?", uuid.Nil, id).Exec()
```

Comment:
```suggestion
	return tx.RawQuery("DELETE FROM "+(&pop.Model{Value: RefreshToken{}}).TableName()+" WHERE user_id = ?",  id).Exec()
```

Should we remove  the check for instance_id?

Reply:
Nope, as I wrote in the description this is intentional, as the database _may_ contain refresh tokens for instances which are not the 0-UUID instance (which is what GoTrue has been using).

When we remove the column from the database (i.e. we are sure there are no other instances in the system) then we can remove the checks.

Reply:
Ah apologies, missed the description. Thanks for the explanation!